Lib/asyncio/constants.py

```diff
@@ -5,3 +5,8 @@
 
 # Seconds to wait before retrying accept().
 ACCEPT_RETRY_DELAY = 1
+
+# Number of stack entries to capture in debug mode.
+# The large the number, the slower the operation in debug mode
+# (see extract_stack() in events.py)
```

Comment:
You may document that this value is passed to the limit parameter of traceback.extract_stack(), or maybe just document that None means "no limit".